airflow/bin/cli.py

```diff
@@ -339,6 +339,16 @@ def run(args, dag=None):
         settings.configure_vars()
         settings.configure_orm()
 
+    # Get task attempt that will be used to separate log based on
+    # each attempt number
+    TI = TaskInstance
+    session = settings.Session()
+    ti = session.query(TI).filter(
```

Comment:
Don't access the database directly. Use models.py in order to
Make sure you load the right ti.